config.yaml

```diff
@@ -178,6 +178,15 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-1.9.3.3
```

Comment:
This belongs into the "mirror" names, see for example https://github.com/will-b/n98-magerun/blob/9f06a0c1860f748607404cb114d1b7edb680ff37/config.yaml#L352